src/components/Pages/Opportunities/Opportunities.js

```diff
@@ -0,0 +1,112 @@
+import React, { useState, useEffect } from 'react'
+import styled from 'styled-components/macro'
+import Project from '../../Project/Project'
+import TagCluster from '../../TagCluster/TagCluster'
+import { sortByTags, filterBySearch } from '../../../services/utils'
+import Search from '../../Search/Search'
+import Overlay from '../../Project/ProjectOverlay/Overlay'
+import Header from '../../Header/Header'
+
+export default function App({ onBack }) {
```

Comment:
Why is this called `App` as well? Shouldn't this component be named `Opportunities`?